Source/Blazorise/Components/Table/TableRowHeader.razor.cs

```diff
@@ -42,14 +43,29 @@ protected override void BuildClasses( ClassBuilder builder )
     /// <inheritdoc/>
     protected override void BuildStyles( StyleBuilder builder )
     {
-        if ( FixedPosition == TableColumnFixedPosition.Start && fixedPositionStartOffset.HasValue )
+        if ( FixedPosition == TableColumnFixedPosition.Start )
         {
-            builder.Append( $"left:{fixedPositionStartOffset:G29}px" );
+            if ( ParentTable.FixedColumnsSync )
+            {
+                var startOffset = ParentTableRow.GetTableRowHeaderFixedPositionStartOffset( this );
+                builder.Append( $"left:{startOffset:G29}px" );
+            }
+            else if ( fixedPositionStartOffset.HasValue )
+            {
+                builder.Append( $"left:{fixedPositionStartOffset:G29}px" );
```

Comment:
We need to consider having a "number to string" utility extension method that converts number to proper string value.

Reply:
ANy ideas for naming? **FormatAs**?

Reply:
- ToPreciseString
- ToG29String

Reply:
What's a "Precise" string :D

Reply:
The G29 "compacts" the number without rounding it. So I think precise makes sense.

Reply:
Ahh. Yea I like that descriptive name better. ToG29String would make no difference at all, it would still be obscure.